src/pudl/metadata/fields.py

```diff
@@ -3289,6 +3350,10 @@
         "description": "Whether each generator record is for one owner or represents a total of all ownerships.",
         "constraints": {"enum": ["owned", "total"]},
     },
+    "ownership_percentage": {
+        "type": "string",
+        "description": "Percentage of subsidiary company owned by parent.",
+    },
```

Comment:
Why is ownership percentage a string instead of a number?

We have an analogous field in [the EIA 860 ownership tables](https://catalystcoop-pudl.readthedocs.io/en/nightly/data_dictionaries/pudl_db.html#core-eia860-scd-ownership) which is `fraction_owned` -- in general we have avoided using percentages (numbers from 0 to 100) and used fractions, since they can be used directly in calculations.

The existing field description is specific to generators, but you can overload it with a resource-specific field description.

Reply:
These were a bit messy and I think we just hadn't had the time to clean them fully to get into floats, but I took a crack at it and it wasn't too hard. These are now floats and using the `fraction_owned` field name with custom descriptions.

Reply:
I had to make them strings because it was hard to clean out all the errors to make it a float within the budget constraints. Also, I was having pydantic issues when I tried to make it a float. But hopefully Zach has fixed that?